en/advanced_config/bootloader_update.md

```diff
@@ -0,0 +1,27 @@
+# Bootloader Update
+
+Early [Pixhawk-series](../flight_controller/pixhawk_series.md#fmu-versions) flight controllers based on FMUv2 had a hardware issue ([Silicon Errata](../flight_controller/silicon_errata.md#fmuv2--pixhawk-silicon-errata)) that restricted them to using 1MB of flash memory. 
+The problem was fixed on newer hardware, so it can now (in theory) install FMUv3 Firmware and access all 2Mb available memory.
+
+Unfortunately some boards come from the factory with an old bootloader that is unable to detect whether or not the hardware issue is present. 
+As a result, the memory-restricted FMUv2 Firmware must still be used.
+
+This topic explains how you can update the bootloader to the latest version so that you can use FMUv3 Firmware on compatible boards.
+
+
+### Main Steps
+
+You can initiate bootloader update on next restart by [setting the parameter](../advanced_config/parameters.md#parameter-configuration): [SYS_BL_UPDATE](../advanced_config/parameter_reference.md#SYS_BL_UPDATE).
+
+To update the bootloader:
+
+1. Insert an SD card (enables boot logging to debug any problems.)
+1. Flash master of `fmu-v2 target` (e.g. through QGC)
```

Comment:
@bkueng This is just master selected from QGC after checkign **Advanced settings** in the firmware update right? ie. QGC selects that it will install the FMUv2, not you right?

Reply:
2x correct